jaq/src/cli.rs

```diff
@@ -106,6 +107,10 @@ impl Cli {
                 let (name, value) = parse_key_val("--arg", args)?;
                 self.arg.push((name, value.into_string()?));
             }
+            "argjson" => {
+                let (name, value) = parse_key_val("--argjson", args)?;
+                self.argjson.push((name, value.into_string()?));
+            }
```

Comment:
As i understand code i cli just collects stings, reading files, parsing etc should happen in binds?